assets/src/scripts/components/hydrograph/store.spec.js

```diff
@@ -97,13 +98,17 @@ describe('Redux store', () => {
         it('should handle SET_MEDIAN_STATISTICS', () => {
             expect(timeSeriesReducer({}, {
                 type: 'SET_MEDIAN_STATISTICS',
-                medianStatistics: 'test data'
+                medianStatistics: {beginYear: '2000', endYear: '2010', values: ['a']}
```

Comment:
Now that there's extra data on `medianStatistics`, I think it may be clearer on the action to flatten it like:

```
{
    type: 'SET_MEDIAN_STATISTICS',
    beginYear: '2000',
    endYear: '2010',
    values: ['a']
}
```

Although I've also seen this convention used in quite a few places, maybe we should do this everywhere...?

```
{
    type: 'SET_MEDIAN_STATISTICS',
    payload: {
        beginYear: '2000',
        endYear: '2010',
        values: ['a']
    }
}
```

Perhaps just ignore this for comment for now, and maybe we can discuss conventions like that after using this for awhile.

Reply:
Is it necessary to store begin and end years on the state, or could we derive that from the date ranges of the median state values?

Reply:
The median values that we get make from the stats service is only a month and date without a year (though begin and end year are specified for each record in the returned RDB). I guess an alternative would be to include the begin and end year in with each of the median values... though having the begin and end year repeated 366 times in the values array seems a bit unnecessary.

Reply:
I end up having to put the current or previous year on the dates, so they will graph properly. So, if we were to use just the dates, we'd end up with things like `2017 - 2018`.

Reply:
Oh, right...

Reply:
Hmm... maybe median stats shouldn't be stored under tsData.

Reply:
Per your other comment, I think we should think about how to structure the redux store.